assets/general/lisk/info.json

```diff
@@ -27,19 +27,32 @@
     "oldPendingAttempts": 4
   },
   "txConsistencyMaxTime": 60000,
-  "nodes": [
-    {
-      "url": "https://lisknode5.adamant.im",
-      "alt_ip": "http://38.242.243.29:44099"
-    }
-  ],
+  "nodes": {
+      "list": [
+          {
+              "url": "https://lisknode5.adamant.im",
+              "alt_ip": "http://38.242.243.29:44099"
+          }
+      ],
+      "additionalInfo": {
+          "normalUpdateInterval": 270000,
+          "crucialUpdateInterval": 30000,
+          "onScreenUpdateInterval": 10000,
+          "threshold": 5
+      }
+  },
   "services": {
     "lskService": [
       {
         "url": "https://liskservice5.adamant.im",
         "alt_ip": "http://38.242.243.29:44098"
       }
-    ]
+    ],
+    "additionalInfo": {
+        "normalUpdateInterval": 330000,
+        "crucialUpdateInterval": 30000,
+        "onScreenUpdateInterval": 10000
+    }
```

Comment:
Every key of that object represent a separate service. By adding `additionalInfo` to that object we may accidentally treat it as a service.

To avoid bugs in the future, I suggest to use similar structure, by wrapping it into `services.list`, as you did for `nodes`:

```json
"services": {
    "list": {
       "lskService": [
          {
             "url": "https://liskservice5.adamant.im",
              "alt_ip": "http://38.242.243.29:44098"
          }
        ]
    },
    "additionalInfo": {
      // ...
    }
```